src/zjs_fs.json

```diff
@@ -1,17 +1,22 @@
 {
     "module": "fs",
     "require": "fs",
-    "targets": ["arduino_101"],
+    "targets": ["arduino_101", "frdm_k64f"],
     "zephyr_conf": {
         "all": [
             "CONFIG_FILE_SYSTEM=y",
             "CONFIG_FILE_SYSTEM_FAT=y",
-            "CONFIG_DISK_ACCESS_FLASH=y",
             "CONFIG_FLASH=y",
             "CONFIG_SPI=y",
             "CONFIG_GPIO=y"
         ],
-        "arduino_101": ["CONFIG_FS_FAT_FLASH_DISK_W25QXXDV=y"]
+        "arduino_101": [
+            "CONFIG_DISK_ACCESS_FLASH=y",
+            "CONFIG_FS_FAT_FLASH_DISK_W25QXXDV=y"
+        ],
+        "frdm_k64f": [
+            "CONFIG_DISk_ACCESS_RAM=y"
```

Comment:
That lowercase k seems surprising. :)

Does this work? It uses a RAMDISK for "flash"?